taggit/managers.py

```diff
@@ -309,13 +339,18 @@ def __lt__(self, other):
         return False
 
     def post_through_setup(self, cls):
-        self.related = RelatedObject(cls, self.model, self)
+        if RelatedObject is not None:  # Django < 1.8
```

Comment:
related.py is gone in the latest code: https://github.com/django/django/tree/stable/1.8.x/django/db/models

Maybe it's a ``.pyc`` file hanging around, could you try reinstalling Django?